commit 440cf2bc1097aee4045337f83d98b97eee1846b2 (from 567291e7347a2545650d9f98a79427dddab8cd22)
Merge: 567291e bd33cf4
Author: ErikZalm <erik@vdzalm.eu>
Date:   Tue May 21 09:36:14 2013 -0700

    Merge pull request #488 from MStohn/fix-typo
    
    fixed typo / added "const" to defines for newer gcc compatibility

diff --git a/Marlin/DOGMbitmaps.h b/Marlin/DOGMbitmaps.h
index a5d8702..984f421 100644
--- a/Marlin/DOGMbitmaps.h
+++ b/Marlin/DOGMbitmaps.h
@@ -1,7 +1,7 @@
 #define START_BMPWIDTH 	60	//Width in pixels
 #define START_BMPHEIGHT 	64	//Height in pixels
 #define START_BMPBYTEWIDTH 	8	//Width in bytes
-unsigned char start_bmp[574] PROGMEM = { //AVR-GCC, WinAVR
+const unsigned char start_bmp[574] PROGMEM = { //AVR-GCC, WinAVR
 0xFF,0xFF,0xFF,0xFF,0xFF,0xFF,0xFF,0xF0,
 0xFF,0xFF,0xFF,0xFF,0xFF,0xFF,0xFF,0xF0,
 0xFF,0xFF,0xFF,0xF9,0xFF,0xFF,0xFF,0xF0,
@@ -71,7 +71,7 @@ unsigned char start_bmp[574] PROGMEM = { //AVR-GCC, WinAVR
 #define STATUS_SCREENWIDTH 		115	//Width in pixels
 #define STATUS_SCREENHEIGHT 	19	//Height in pixels
 #define STATUS_SCREENBYTEWIDTH 	15	//Width in bytes
-unsigned char status_screen0_bmp[] PROGMEM = { //AVR-GCC, WinAVR
+const unsigned char status_screen0_bmp[] PROGMEM = { //AVR-GCC, WinAVR
 0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x7F,0xFF,0xE0,
 0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x70,0x00,0xE0,
 0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x63,0x0C,0x60,
@@ -96,7 +96,7 @@ unsigned char status_screen0_bmp[] PROGMEM = { //AVR-GCC, WinAVR
 #define STATUS_SCREENWIDTH 		115	//Width in pixels
 #define STATUS_SCREENHEIGHT 	19	//Height in pixels
 #define STATUS_SCREENBYTEWIDTH 	15	//Width in bytes
-unsigned char status_screen1_bmp[] PROGMEM = { //AVR-GCC, WinAVR
+const unsigned char status_screen1_bmp[] PROGMEM = { //AVR-GCC, WinAVR
 0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x7F,0xFF,0xE0,
 0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x70,0x00,0xE0,
 0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x61,0xF8,0x60,
diff --git a/Marlin/dogm_lcd_implementation.h b/Marlin/dogm_lcd_implementation.h
index c45a6a6..1744e60 100644
--- a/Marlin/dogm_lcd_implementation.h
+++ b/Marlin/dogm_lcd_implementation.h
@@ -19,7 +19,7 @@
 * Implementation of the LCD display routines for a DOGM128 graphic display. These are common LCD 128x64 pixel graphic displays.
 **/
 
-ifdef ULTIPANEL
+#ifdef ULTIPANEL
 #define BLEN_A 0
 #define BLEN_B 1
 #define BLEN_C 2

